Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -676,6 +684,11 @@ - (void)viewWillAppear:(BOOL)animated
     }
 
     [self updateLastVisibleTimestamp];
+
+    if (!self.viewHasEverAppeared) {
+        NSTimeInterval appearenceDuration = CACurrentMediaTime() - self.viewControllerCreatedAt;
+        DDLogInfo(@"%@ First viewWillAppear took: %.2fms", self.logTag, appearenceDuration * 1000);
+    }
```

Comment:
Did you mean to commit this?  If you want to leave this in, can we make it "verbose"?

Reply:
I did mean to keep it. I'll reduce the log level.